Makefile

```diff
@@ -55,3 +55,7 @@ site: html
 # Copy changes to the repo from which they are served
 gh-pages: site
 	python _scripts/gh-pages.py
+
+team:
+	python tools/team_list.py > team.rst
```

Comment:
What do you think about (re)creating the `team.rst` with each website build in the CI?

Reply:
Sorry, my bad. Just noticed that updating the website isn't fully automated anyway. :sweat_smile:

Reply:
@lagru Perfectly fine; we must just set up the tokens.  With me, I mean maybe... some volunteer ;)

Reply:
Noted. I'll add it to my idea list. :smile: